src/Orikivo/Services/TemplateBuilder.cs

```diff
@@ -1,512 +1,378 @@
 ﻿using System;
+using System.Collections.Generic;
 using System.Drawing;
 using System.Linq;
+using Microsoft.VisualBasic;
 
 namespace Orikivo.Systems.Services
 {
     public class TemplateBuilder
     {
+        const int iconPos = 0;
+        const int largeLimit = 17;
+        const int mediumLimit = 22;
+        const int smallLimit = 31;
+        const string baseProfileTemplate = ".//resources//profile//template.png";
+        const string statusType = ".//resources//profile//status.png";
+        const string iconSet = ".//resources//profile//icons.png";
+        const string currencyAmountSet = ".//resources//profile//currency-symbols.png";
+
         public static Bitmap BuildNewLevelTemplate(Bitmap userAvatar)
         {
-
             const string baseLevelTemplate = ".//Templates//lvl_wide.png";
-
             var template = Image.FromFile(baseLevelTemplate);
             var avatar = userAvatar;
             var templateBuilder = new Bitmap(template.Width, template.Height, template.PixelFormat);
-            using (var graphic = Graphics.FromImage(templateBuilder))
-            {
-                var templateSize = new Rectangle(0, 0, template.Width, template.Height);
-                graphic.DrawImage(template, templateSize);
-                template.Dispose();
-                var avatarPosition = new Point(16, 16);
-                graphic.DrawImage(avatar, avatarPosition);
-                avatar.Dispose();
-                return templateBuilder;
-            }
+            using var graphic = Graphics.FromImage(templateBuilder);
+            var templateSize = new Rectangle(0, 0, template.Width, template.Height);
+            graphic.DrawImage(template, templateSize);
+            template.Dispose();
+            var avatarPosition = new Point(16, 16);
+            graphic.DrawImage(userAvatar, avatarPosition);
+            avatar.Dispose();
+            return templateBuilder;
         }
 
-        public static Bitmap BuildNewProfileBaseTemplate
```

Comment:
You can leave the method parameters listed out individually, and instead separate the last three parameters that are on the same line for easier readability.